ChemoTracker/components/Profile/ProfileEdit.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import { Text, View, Image, ScrollView, TouchableWithoutFeedback, Button } from 'react-native';
+import { TabNavigator } from 'react-navigation';
+import CategoryCell from './Cell.js';
+import styles from '../../styles/profile-main.js';
+import color from '../../styles/color.js';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import IonIcon from 'react-native-vector-icons/Ionicons';
+import { Form, Separator,InputField, LinkField, SwitchField, PickerField, DatePickerField, TimePickerField} from 'react-native-form-generator';
+
+class ProfileEdit extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      formData: null
+    }
+  }
+
+  dismissView = () => {
+    console.log("dismissView");
+    this.props.navigation.goBack();
+  }
+
+  saveChanges = () => {
+    console.log("save changes");
+    this.props.navigation.goBack();
+  }
+
+  static navigationOptions  = ({ navigation }) => {
```

Comment:
Maybe move this to ProfileTab? I'm assuming that's where all your navigation logic is?

Reply:
This needs to be here bc the event function for the header should be done here not in Profile tab